internal/verify/cosign/client.go

```diff
@@ -0,0 +1,39 @@
+package cosign
+
+import (
+	"context"
+	"crypto"
+
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/sigstore/cosign/pkg/cosign"
+	"github.com/sigstore/cosign/pkg/oci"
+	"github.com/sigstore/sigstore/pkg/cryptoutils"
+	"github.com/sigstore/sigstore/pkg/signature"
+)
+
+type Verifier struct {
```

Comment:
This needs documentation. What does this struct do?

Reply:
We removed it.